src/panels/Scene/SceneTree/FeaturedSceneTree.tsx

```diff
@@ -57,7 +57,7 @@ export function FeaturedSceneTree() {
         data={featuredTreeData}
         renderNode={(payload) => <SceneTreeNode {...payload} />}
       />
-      <Divider my={'xs'} />
+      <Divider py={'xs'} />
```

Comment:
Unfortunately, it seems like the divider does not really support padding. This removed the spacing around the line...

 Try bringing back the margin? Did it cause problems?

Reply:
I made a fix for this using the `Space` component